### PR TITLE
feat(icon): add dialpad16 icon

### PR DESCRIPTION
**Pull request by yvniTop**
[CT-2094]

### Description

Added Dialpad16 icon.

### How to test

- FIXME: Add the steps describing how to verify your changes

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| Insert screenshots or screen recordings |  |

### Review

- [x] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/api-principles.md)
- [x] Annotate all `props` in component with documentation
- [ ] Create `examples` for component
- [x] Ensure that deployed demo has expected results and good examples
- [x] Ensure that tests pass by running `yarn test`
- [ ] Ensure that visuals tests pass by running `yarn test:visual`. If not - check the documentation [how to fix visual tests](https://github.com/toptal/picasso/blob/master/docs/contribution/visual-testing.md#fixing-broken-visual-tests-inside-a-pr)
- [x] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

[CT-2094]: https://toptal-core.atlassian.net/browse/CT-2094?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by dmaklygin1 — APPROVED** [review]

File: packages/picasso/src/Icon/Dialpad16.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import React, { forwardRef, Ref } from 'react'
```
Comment (OleksandrNechai) [inline packages/picasso/src/Icon/Dialpad16.tsx:1]: Hey! Thanks for your contribution!
I don't see this icon in BASE documentation. https://app.abstract.com/projects/1b06c884-06af-482a-bf12-a82f521a19a1/branches/master/files/2fbda9fd-63f3-42e7-a9a8-135a6562ac01?s=dial

We either need designers to add it or this icon should go to the respective project as project-specific and not useful for other Picasso users.

**Review by OleksandrNechai — CHANGES_REQUESTED** [review]
Icon seems to be missing in BASE docs

File: packages/picasso/src/Icon/Dialpad16.tsx (hunk 1 above)
Comment (yvniTop) [reply packages/picasso/src/Icon/Dialpad16.tsx:1]: Hey @OleksandrNechai , isn't it that one? https://app.abstract.com/projects/1b06c884-06af-482a-bf12-a82f521a19a1/branches/master/commits/93003c4321691ec9a25851a46b6bc247f88aa2c6/files/2fbda9fd-63f3-42e7-a9a8-135a6562ac01/layers/00EEBF49-F0F0-454C-B616-F744B8796629?mode=build&selected=158798760-83BB22BF-61F3-45B9-A3B1-A6CA9E07FA22

Comment (TomasSlama) [reply packages/picasso/src/Icon/Dialpad16.tsx:1]: image
yep it is there

**Review by OleksandrNechai — APPROVED** [review]

**Pull request merged**